### PR TITLE
Update globalsearch.php

### PR DESCRIPTION
**Pull request by kizule**
Removed font-size element so title of each namespace can be viewed better.

See pictures (before) https://prnt.sc/o5ow9y (and after fix) https://prnt.sc/o5ow1c

### CODE REVIEW

**Comment by kizule** [comment]
@samwilson Can you merge this please?

File: globalsearch.php [hunk 1]
```diff
@@ -86,7 +86,6 @@
 								<input id='searchfor' name='searchfor' size='30' type='td' value=''>
 							</td>
 						</tr>
-						<tr style='font-size:0.5em;'>
```
Comment (samwilson) [inline globalsearch.php:89]: Only the `style` attribute should be removed, not the whole element. The `<tr>` is part of the table structure.

**Review by samwilson — CHANGES_REQUESTED** [review]
Note that I'm not a maintainer of this repo. Happy to help review though.

File: globalsearch.php (hunk 1 above)
Comment (kizule) [reply globalsearch.php:89]: Should be ok now, check it please.

**Review by samwilson — APPROVED** [review]
Looks good. @davidbarratt are you a maintainer here?

**Pull request merged**